caikit/interfaces/nlp/data_model/classification.py

```diff
@@ -148,6 +148,8 @@ class TextGenTokenClassificationResults(DataObjectBase):
     warnings: Annotated[
         Optional[List[InputWarning]], FieldNumber(9)
     ]  # Warning to user in the event of input errors
+    tokens: Annotated[Optional[List[GeneratedToken]], FieldNumber(11)]
```

Comment:
q - do 10 and 11 actually conflict here? Wondering if (or how) this is different from `GeneratedToken` below since field 3 is present there and in the classes that use it?

Reply:
I believe the content of the 2 fields is different:

`output` stores the HAP evaluation of the sentence.
```
        "output": [
            {
                "start": 0,
                "end": 35,
                "word": "All locals are rude to foreigners.",
                "entity": "has_HAP",
                "entity_group": "",
                "score": 0.9498854279518127,
                "token_count": None,
            }
        ],
```

while the `tokens` field stores the individual token and the associated logprob value:
```
    "tokens": [
        {"text": "Ġmuch", "logprob": 0.0},
        {"text": "Ġthey", "logprob": 0.0},
        {"text": "Ġhave", "logprob": 0.0},
        {"text": "Ġbeen", "logprob": -0.693359375},
        {"text": "Ġgiven", "logprob": -0.693359375},
        {"text": "Ġand", "logprob": -0.693359375},
        {"text": "Ġwhat", "logprob": 0.0},
        ...
    ],
```

Is this what you are referring to, or is it something different?

Reply:
I'm wondering if the actual field numbers can be 10 and 11 as opposed to 11 and 12 here?